packaging/standalone/src/main/distribution/shell-scripts/bin/neo4j-arbiter

```diff
@@ -273,81 +121,41 @@ showinfo() {
   exitonnojava
   buildclasspath
 
-  echo "NEO4J_HOME:        $NEO4J_HOME"
-  echo "NEO4J_INSTANCE:    $NEO4J_INSTANCE"
-  echo "JAVA_HOME:         $JAVA_HOME"
-  echo "JAVA_OPTS:         $JAVA_OPTS"
-  echo "CLASSPATH:         $CLASSPATH"
+  echo "NEO4J_HOME:        ${NEO4J_HOME}"
+  echo "JAVA_HOME:         ${JAVA_HOME}"
+  echo "JAVA_OPTS:         ${JAVA_OPTS}"
+  echo "CLASSPATH:         ${CLASSPATH}"
 }
 
-# END FUNCTIONS
-# BEGIN MAIN
-
-# Parse option flags
-while getopts "u:h" opt; do
-  case $opt in
-    h)
-      echo "Running in headless (-h) mode" >&2
-      HEADLESS=true
-      ;;
-    u)
-      echo "Installing with user $OPTARG"
-      NEO4J_USER=$OPTARG
-      ;;
-    \?)
-      echo "Invalid option: -$OPTARG" >&2
-      exit 1
-      ;;
-    :)
-      echo "Option -$OPTARG requires an argument." >&2
-      exit 1
-      ;;
-  esac
-done
-
-case "${!OPTIND}" in
+case "$1" in
   console)
     console
-    exit 0
     ;;
 
   start)
     startit
-    exit 0
     ;;
 
   stop)
     stopit
-    exit 0
     ;;
 
   restart)
     stopit
     startit
-    exit 0
     ;;
 
   status)
     reportstatus
-    exit 0
     ;;
 
   info)
     showinfo
-    exit 0
-    ;;
-  install)
-    installservice
-    exit 0
     ;;
-  remove)
-    removeservice
-    exit 0
-    ;;
-  *)
-    echo "Usage: neo4j { start | stop | restart | status | info | install | remove }"
-    exit 1;;
 
+  *)
+    echo "Usage: neo4j { console | start | stop | restart | status | info }"
```

Comment:
A lof of this script looks *very* close to the neo4j one. Can we have this one source that one, and just add its weirdities? Or are they not close enough?